desk/app/chat.hoon

```diff
@@ -2240,25 +2252,28 @@
       (di-give-writs-diff diff)
     ::
         %del
-      =?  di-core  ?=(^ had)
-        =*  content  content.writ.u.had
+      =?  di-core  &(?=(^ had) ?=(%& -.writ.u.had))
+        =*  content  content.+.writ.u.had
```

Comment:
Here and elsewhere, if there isn't a face on there, you don't need the lark. Name resolution will just walk the tree to the first instance of the face you ask for.

```suggestion
        =*  content  content.writ.u.had
```